.github/workflows/yocto.yml

```diff
@@ -31,10 +31,14 @@ jobs:
       with:
         mode: build
 
+    - name: Get the number of new commits in 24 hours
+      run: echo "NEW_COMMIT_COUNTS=$(git log --oneline --since 24.hours | wc -l)" >> $GITHUB_ENV
```

Comment:
I didn't know there is an option like `--since 24.hours`.
I can change this long shell script to just one line using above option at NNS_Setup 👍 
```sh
pushd nnstreamer-orig
git log -1 --format="%s" > ${{ github.workspace }}/orig
popd
pushd nnstreamer-mirror
git log -1 --format="%s" > ${{ github.workspace }}/mirror
popd
if ! cmp orig mirror; then
  echo "createPR=1" >> "$GITHUB_ENV"
else
  echo "createPR=0" >> "$GITHUB_ENV"
fi
```